queries.sql

```diff
@@ -184,3 +184,4 @@ WHERE animals.name = 'Maisy Smith'
 GROUP BY species.name
 ORDER BY COUNT(*) DESC
 LIMIT 1;
+
```

Comment:
- [ ] Kindly add the used insert statement to `data.sql` file as per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md#:~:text=If%20you%20decide%20to%20commit%20any%20INSERT%20INTO%20queries%20%2D%20remember%20to%20add%20the%20m%20to%20the%20data.sql%20file).